app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddPitchSurface.kt

```diff
@@ -0,0 +1,56 @@
+package de.westnordost.streetcomplete.quests.surface
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.meta.ALL_ROADS
+import de.westnordost.streetcomplete.data.meta.updateWithCheckDate
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+
+
+class AddPitchSurface : OsmFilterQuestType<SurfaceAnswer>() {
+    private val sportValuesWherePitchSurfaceQuestionIsInteresting = listOf(
+        "soccer", "tennis", "basketball", "equestrian", "athletics", "volleyball", "skateboard",
+        "bmx", "american_football", "rugby_union", "badminton", "pelota", "horse_racing",
+        "disc_golf", "futsal", "cycling", "gymnastics"
+    )
+
+    override val elementFilter = """
+        ways with leisure=pitch
+        and sport ~ ${sportValuesWherePitchSurfaceQuestionIsInteresting.joinToString("|")}
```

Comment:
```suggestion
        and sport ~ .*${sportValuesWherePitchSurfaceQuestionIsInteresting.joinToString("|")}.*
```
... to catch pitches with multiple sports like `sport=soccer;basketball`?